weekly-standup-template.md

```diff
@@ -41,32 +41,27 @@ Current repos are:
 ### Things that are blocking me
 
 <!--
-Things you'd like to discuss in person with other team members
+Please use the template below to report things that are blocking you.
+This may be things out of your control that you need to raise attention to.
+It may also be things you would like the help or input of the team on.
 -->
 
-### Pans for the week
-Please include full details for Stories or issues, either link or as org/repo#number
-Current repos are:
-- alan-turing-institute/data-safe-haven-team#X
-- alan-turing-institute/data-safe-haven#X
-- alan-turing-institute/trusted-research#x
-- sa-tre/satre-specification#X
-- sa-tre/satre-team#X
 <!--
 #### Your name here
 
-- Work on Story Z
-- by addressing issues X
+- I don't understand bug X
+- I don't feel I have the authority to do Y
+- I want advice on issue Z
 -->
 
 ## Synchronous items
 
 ### Agenda
 
 1. Address blockers
-2. Plan coworking sessions
-3. Consider prioritisation (potentially assign new issues)
```

Comment:
Is the idea here to help address the difference between 'the plan' and 'what we are actually doing'?

Is this meeting the right place for this? I want to try and 'protect' people from project management discussions if they don't need to be involved. I think before that came up as a problem with the weekly meetings, too much high level discussion, felt like a waste of time.

This is the sort of thing I was hoping to cover by adding the two new actions for PM and technical lead.

Reply:
I think there's smth useful in, if new work is suggested/brought up, it still aligns with overall team strategy and direction, and just doing an explicit sense check of this. The danger otherwise is people picking up random pieces of work that don't easily fit together. I don't think it needs to be a long convo, just the person bringing the new work in to justify how it fits with our wider team vision

Reply:
Whatever we decide with this I think we should also have some cadence check-in of ensuring we're still on track to our vision with the work we are doing (maybe this is monthly meeting)

Reply:
OK, I think I understand now.
Agreed that being careful and deliberate about when we take on new work is important.

Do we think this isn't captured in "What I've been doing"? Things should stand out there if they are not attached to a task/story. I feel there might be a better, asynchronous way to flag this kind of thing.

I would be interested to hear from @jemrobinson, @craddm and @edwardchalstrey1 here.

Reply:
I wonder if the monthly meeting is a better place to bring new proposed pieces of work (are they called stories?). Then we can keep this meeting focused on what the DSH team are doing. A related question - should TRESA come to this meeting too or do we want to maintain separation between the two teams?

Reply:
I think separate as we are eventually going to separate them... but the head of TRESA should come @jemrobinson

Reply:
- Task level work should be addressed in this meeting, verify it matches exisiting stories (yup that's the name)
- New stories should be addressed in monthlies
- @JimMadge I can agree with highlighting new work as part of "What I have being doing?" or "plans for the week" but let's make sure it is obvious (no one will know the others list of issue by heart, and checking them all can take time)

Reply:
I am unsure of the separation of DSH and TRESA while the resources are shared... and where does the separation exactly lies? there is work I do from my DSH time for TRESA

Reply:
OK, I think I have addressed these ideas in the last few commits.

Regarding TRESA. I suppose I don't object to TRESA sitting in, but I don't want to overcomplicate this meeting by adding more items, sections for TRESA.

I think it is already on the border of being too much busy work.